dags/config/config.yaml

```diff
@@ -102,3 +117,4 @@ cmf:
     top-level: 2_Active_Data
     admin: 202_adm
     roads: 232_tran
+    rail: 242_tran  # may not fit into naming convention!!!
```

Comment:
One tiny thing, the folder `232_tran` is for all transport data, so includes roads and rail. Hence the correct value here is:
`rail: 232_tran`